### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by pepicrft**
Following @cpisciotta's suggestion, I'm adding a `CODEOWNERS` file to notify him about relevant changes in this repository.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Sources/** @cpisciotta
```
Comment (pepicrft) [inline .github/CODEOWNERS]: @cpisciotta let me know if you'd change the granularity

**Review by cpisciotta — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (cpisciotta) [reply .github/CODEOWNERS]: This should be fine 👍

**Comment by cpisciotta** [comment]
@pepicrft You might also consider introducing a GitHub team instead of defining individuals in this file. I'm not sure who else (if anyone) is actively involved in maintaining this project.

**Comment by pepicrft** [comment]
> You might also consider introducing a GitHub team instead of defining individuals in this file. I'm not sure who else (if anyone) is actively involved in maintaining this project.



> (if anyone) is actively involved in maintaining this project.

I've added the @tuist/core team as well.

**Pull request merged**